MusicNotationKit/MusicNotationKit/NoteDuration.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  NoteDuration.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 8/20/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct NoteDuration {
```

Comment:
Should there be a method that calculates the note duration taking into account the dotCount?   Since the dotCount doesn't have a max limit, the method would probably have to return a fraction value, for example 1/2 + 1/4 = 5/4. This is were TPQN would be nice, because it removes the need to work with fractions.